### DISCUSSION TITLE
How to test Modules

### DISCUSSION THREAD

**Discussion by lilmart210** [tradingview/lightweight-charts#1138] [General] [answered]
I am interested in extending features of this library and implementing things i want. But i am new to github and js libraries in general. Is there some video i could watch to understand how to go about doing this?

**Comment by SlicedSilver** [comment, marked as answer]
Learning about GitHub and developing JS libraries will take some time and would require watching more than one video. The internet is full of excellent resources and I would recommend that you follow an online course (instead of a single video). Perhaps look to one of the following sites (from a quick google search):
- freeCodeCamp
- Codecademy
- Udemy
- Frontend masters
